lib/compilers/rust.ts

```diff
@@ -222,6 +223,14 @@ export class RustCompiler extends BaseCompiler {
         return options;
     }
 
+    override optionsForDemangler(filters?: ParseFiltersAndOutputOptions): string[] {
```

Comment:
This seems like something that could (maybe) be lifted into the baseCompiler so other compilers could take advantage of, but I see that this is very specific to `c++filt` and `--format=rust`? I see it's copy/pasted in gccrs.ts too but as it's only 6 lines I think this is probably ok.

Reply:
I also thought about that, but it seemed to make the code more complex. If you’d like, I can make a PR to add a new field to `CompilerInfo` to set the `--no-verbose` flag in the config files? Or to deduplicate the copy/pasted code?

As eddyb said, `--no-verbose` is also available for C++ (and presumably also for the other mangling formats `c++filt` supports?). I tried to find a case in C++ where `--no-verbose` makes a difference, but I couldn’t find one.